include/OpenSpaceToolkit/Astrodynamics/Trajectory/Sequence.hpp

```diff
@@ -147,19 +146,21 @@ class Sequence
 
     void addManeuverSegment(const Shared<EventCondition>& anEventConditionSPtr, const Shared<Thruster>& aThruster);
 
+    /// @brief                  Solve the sequence given an initial state, for a number of reptitions.
+    ///
+    /// @param                  [in] aState Initial state for the sequence.
+    /// @param                  [in] aRepetitionCount Number of repetitions. Defaults to 1.
```

Comment:
Maybe add `Default to 1 (1 meaning that the sequence is run once)` in case somebody wonders "does 1 mean that it repeats itself once and therefore the total number of runs id 2?"